src/cookbooks/bigquery.md

```diff
@@ -7,10 +7,11 @@ BigQuery uses a columnar data storage format called [Capacitor](https://cloud.go
 
 There is a cost associated with using BigQuery based on operations. As of right now we pay an on-demand pricing for queries based on how much data a query scans. To minimize costs see [_Query Optimizations_](bigquery.md#query-optimizations). More detailed pricing information can be found [here](https://cloud.google.com/bigquery/pricing).
 
-As we transition to [GCP](https://cloud.google.com) BigQuery has become our primary data warehouse and
-SQL Query engine. Our previous SQL Query
-Engines, Presto and Athena, and our Parquet data lake will no longer be accessible
-by the end of 2019. Specific guidance for transitioning off of the AWS data
+With the transition to [GCP](https://cloud.google.com) in 2019, BigQuery has become our primary data warehouse and
+SQL Query engine.
+Our previous SQL Query Engines, Presto and Athena, and our Parquet data lake will no longer be accessible
+by the end of 2019.
+Specific guidance for transitioning off of the AWS data
```

Comment:
I would say let's just take this section out -- effectively there is no other alternative than bigquery at this point and there is little reason to talk about the migration.

Reply:
I think it may still be worth keeping this info until the end of the year, in case there are late adopters who need help. I will file a follow-up to remove this in the new year.

Reply:
Follow-up issue: https://github.com/mozilla/firefox-data-docs/issues/369